icons/pessoa-com-deficiencia.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this filename should be in english (I think this is the pattern)

Reply:
Yes, but the icon is an image with the letters "PCD", which if we translate the name it will make no sense when someone call it, right? I've talked with @laiseing and she said to keep it in portuguese, so i guess it is ok :joy:

Reply:
I think that icons that will not be usefull to others we should keep them in pec's repository.